ulib/FStar.SizeT.fsti

```diff
@@ -10,17 +10,20 @@ val t : eqtype
 
 val fits (x: nat) : Tot prop
 
+/// According to the C standard, "the bit width of t is not less than 16 since c99"
+/// (https://en.cppreference.com/w/c/types/t)
+
+val fits_at_least_16 (x:nat)
+  : Lemma 
+    (requires x < pow2 16)
```

Comment:
Should this be a `normalize_term (pow2 16)` to simplify its use?

Reply:
I thought about it, but didn't add it, because FStar.UInt is in the dependence graph of FStar.SizeT and UInt provides a lemma with SMT pats for common powers of two, including pow2 16.